server/src/main/java/org/candlepin/resource/PoolResource.java

```diff
@@ -228,7 +233,7 @@ public Pool getPool(@PathParam("pool_id") @Verify(Pool.class) String id,
             "Subscription Pool with ID ''{0}'' could not be found.", id));
     }
 
-    @ApiOperation(notes = "Remove a Pool", value = "deletePool")
+    @ApiOperation(notes = "Remove a Pool", value = "deleteSinglePool")
```

Comment:
I don't understand what this denotes. Also, like the method name in CPM, the "single pool" bit is already expressed by the non-plural noun "pool" and the parameter list that accepts only one pool.